packages/shared/src/components/post/PostModalActions.tsx

```diff
@@ -89,22 +89,24 @@ export function PostModalActions({
 
   return (
     <Container {...props} className={classNames('gap-2', className)}>
-      <SimpleTooltip
-        placement="bottom"
-        content="Read article"
-        disabled={!inlineActions}
-      >
-        <Button
-          className={inlineActions ? 'btn-tertiary' : 'btn-secondary'}
-          tag="a"
-          href={post.permalink}
-          target="_blank"
-          icon={<OpenLinkIcon />}
-          onClick={onReadArticle}
+      {onReadArticle && (
+        <SimpleTooltip
+          placement="bottom"
+          content="Read article"
+          disabled={!inlineActions}
         >
-          {!inlineActions && 'Read article'}
-        </Button>
-      </SimpleTooltip>
```

Comment:
Read article button does not open the post in a new tab

Reply:
I think it should be based on your preference in the settings.

Reply:
No, it does open it on regular posts, just not on shared post

Reply:
Looks like you have fixed it ❤️